### ISSUE TITLE
Boosted Regression Tree SDM pipeline

### ISSUE THREAD

**Issue by jmlord** [GEO-BON/bon-in-a-box-pipelines#168]

**Comment by tpoisot** [comment]
Agreed - @gottacatchenall and I can take the lead on this

**Comment by gottacatchenall** [comment]
Agreed, re https://github.com/GEO-BON/bon-in-a-box-pipelines/pull/99#issuecomment-2170361616, this can also include separating crossvalidation and performance metric computation into separate scripts from the BRT itself

**Comment by jmlord** [comment]
@tpoisot @gottacatchenall 
Do you think this could happen in September? So we have time to merge, re-run examples, feature them on the website by COP16?

(@JoryGriffith and I can assist if need be)

**Comment by tpoisot** [comment]
I think a bare bones version, yes, but also as @gottacatchenall said the next logical step is to provide cross validation as steps for all sdm pipelines to use. This can happen post COP.

**Comment by gottacatchenall** [comment]
> Could we just replace the maxent step with the BRT step in the other pipeline?

Based on a brief look that should be pretty easy. https://github.com/PoisotLab/SpeciesDistributionToolkit.jl/pull/227 (SimpleSDMLayers.jl v1.0) is very close to being merged, so once that's done and on the Julia Pkg registry, I'll work on this

**Comment by JoryGriffith** [comment]
Ideally we would have a working pipeline for the sampling prioritization done for COP16 as well. Do you think that would be feasible once the BRT pipeline is working?

**Comment by tpoisot** [comment]
@JoryGriffith very likely (and I agree about the importance of having it)

**Comment by gottacatchenall** [comment]
> Ideally we would have a working pipeline for the sampling prioritization done for COP16 as well. Do you think that would be feasible once the BRT pipeline is working?

There's a few different options here

1. Simplest -- One of the outputs of the BRT SDM is an map of model uncertainty. At the GEO-BON conference last year we presented this as a priority map for where to sample. We'll definitely have that by COP, hopefully by Thursday when @tpoisot and I will spend few hours on this 

2. Less Simple -- I've been procrastinating a major refactor of BONs.jl (https://github.com/PoisotLab/BiodiversityObservationNetworks.jl/pull/65) until SDMLayers v1.0 was ready. It'd be pretty easy to add an extra pipeline that takes the SDM uncertainty from (1) and the uses an updated version of BONs.jl to sample actual coordinates from it proportional to uncertainty, and getting (pr) merged is something I've planned to do in the next month or so anyway. 

3. Even Less Simple -- With BONs.jl code as is, I've been able to come up with a toy-example of what selecting sites for a national Canadian BON (CAN-BON) would look like. This would be much truer to the spirit/name "BON-in-a-Box", but would require a bit more work than the first two options to port to a pipeline. This might work better in the paper itself though, I've included an example figure in the concept manuscript google-doc

**Comment by jmlord** [comment]
#190 and #195
